### PR TITLE
Add about static pages and model page. 

### PR DESCRIPTION
**Pull request by jonkiky**

### CODE REVIEW

File: src/main/frontend/package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "jest-resolve": "23.6.0",
     "jest-watch-typeahead": "^0.2.1",
     "line-awesome": "github:icons8/line-awesome",
+    "material-design-icons": "^3.0.1",
```
Comment (AjayDoddapaneni) [inline src/main/frontend/package.json]: we don't need this?

Comment (jonkiky) [reply src/main/frontend/package.json]: Yes, In the model page. need icons as buttons.

Comment (AjayDoddapaneni) [reply src/main/frontend/package.json]: You can use the icon as buttons in following manner, https://github.com/CBIIT/icdc-codebase/blob/master/src/main/frontend/src/components/ProfileMenu/ProfileMenuView.js#L31

Comment (jonkiky) [reply src/main/frontend/package.json]: Just removed lib.

**Pull request merged**